pkg/capacityscheduling/README.md

```diff
@@ -0,0 +1,209 @@
+# Overview
+
+This folder holds the capacity scheduling plugin implementations based on [Capacity Scheduling](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/kep/9-capacity-scheduling).
+
+## Maturity Level
+
+<!-- Check one of the values: Sample, Alpha, Beta, GA -->
+
+- [ ] 💡 Sample (for demonstrating and inspiring purpose)
+- [ ] 👶 Alpha (used in companies for pilot projects)
+- [ ] 👦 Beta (used in companies and developed actively)
+- [x] 👨 Stable (used in companies for production workloads)
+
+## Tutorial
+
+Example config:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1alpha2
+kind: KubeSchedulerConfiguration
+leaderElection:
+  leaderElect: false
+clientConnection:
+  kubeconfig: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
+profiles:
+- schedulerName: default-scheduler
+  plugins:
+    preFilter:
+      enabled:
+        - name: CapacityScheduling
+    postFilter:
+      enabled:
+        - name: CapacityScheduling
+      disabled:
+        - name: "*"
+    reserve:
+      enabled:
+        - name: CapacityScheduling
+  pluginConfig:
+    - name: CapacityScheduling
+      # optional plugin configs
+      # default path is /etc/kubernetes/scheduler.conf
+      args:
+        kubeConfigPath: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
```

Comment:
Yes, it will redundant with clientConnection and other plugins which need kubeconfig.
I think we can expose this parameter in upstream for plugins to use. Our internal implementation does this.